lib/classes/Gallery.php

```diff
@@ -0,0 +1,36 @@
+<?php
+namespace Trendwerk\TrendPress;
+
+final class Gallery
+{
+    public function __construct()
+    {
+        add_action('init', [$this, 'replace']);
+    }
+
+    public function replace()
+    {
+        remove_shortcode('gallery');
+        add_shortcode('gallery', [$this, 'add']);
+    }
+
+    public function add($attributes)
+    {
+        if (empty($attributes['ids'])) {
```

Comment:
True, chose not to support this since we never use a gallery without ID's. For a user it would also be unclear where the images would come from. I'd be happy to re-add support, let me know.